dotnet/HtmlformatterTest/Cucumber.HtmlformatterTest.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
+    <PackageReference Include="MSTest" Version="3.6.4" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\HtmlFormatter\Cucumber.HtmlFormatter.csproj" />
```

Comment:
This doesn't look right.

Reply:
The test project is taking a direct project dependency on the primary library. 
It's the conventional way of setting up test project.
See from Messages:
    <ProjectReference Include="..\Cucumber.Messages\Cucumber.Messages.csproj" />